pkg/kclient/kclient.go

```diff
@@ -228,3 +228,19 @@ func (c *Client) IsResourceSupported(apiGroup, apiVersion, resourceName string)
 	}
 	return supported, nil
 }
+
+// GenerateOwnerReference generates an ownerReference which can then be set as
+// owner for various Kubernetes objects and ensure that when the owner object is
+// deleted from the cluster, all other objects are automatically removed by
+// Kubernetes garbage collector
+func GenerateOwnerReference(deployment appsv1.Deployment) metav1.OwnerReference {
```

Comment:
Not sure if this is meant to be here, just to bring it to your attn since odo is using the library generators

Reply:
Fixed :+1: